consensus.go

```diff
@@ -181,6 +181,18 @@ func New(validator SignKey, transport Transport, opts ...ConfigOption) *Pbft {
 	return p
 }
 
+// IsStateLocked only for debug purpose
+// TODO: remove this
+func (p *Pbft) IsStateLocked() bool {
+	return p.state.IsLocked()
+}
+
+// Proposal only for debug purpose
+// TODO: remove this
```

Comment:
Do we still need this?

Reply:
Currently at the end of the test, we are logging locked indication and actual proposal for all the nodes, which is convenient to verify everything is as expected:
`t.Logf("Node %v, isProposalLocked: %v, proposal data: %v\n", n.name, n.pbft.IsStateLocked(), n.pbft.Proposal().Data)`

However, we are thinking through some of the options and not sure what is the best:
1. keep this log and just remove those TODO comments for these two methods,
2. whether it would be better to turn those logs into asserts, where we should check that proposals are different and that required nodes are locked,
3. remove that log within the test, keep `WaitForHeight` and asserting that there is no (timeout) error after invoking it. In that case, we can remove those two methods.

WDYT is the best solution or can you come up something different, more convenient?

Reply:
For now, I think I am fine with 1 to be honest.